source/validate_builtins.cpp

```diff
@@ -753,11 +753,7 @@ spv_result_t BuiltInsValidator::ValidateClipOrCullDistanceAtDefinition(
   }
 
   // Seed at reference checks with this built-in.
-  id_to_at_reference_checks_[inst.id()].push_back(
```

Comment:
Shouldn't the change be to *additionally* check the defining instruction?
Are we losing the checks at the references?  I'm confused

Reply:
ValidateClipOrCullDistanceAtReference will register itself at id_to_at_reference_checks_ on its own.

Reply:
I see, that was pre-existing functionality.  Thanks.